pkg/provision/ubuntu.go

```diff
@@ -155,6 +155,11 @@ WantedBy=multi-user.target
 		return nil, err
 	}
 
+	// because in kic base image we pre-install docker it already has a service file, it is harmless to double check with this step
```

Comment:
I don't want harmless commands to be executed. Docker is a hermetic environment: either we need to run this command or not. Be clear that this has to be run to pick up the reconfigured docker file.